test/err.js

```diff
@@ -27,3 +28,16 @@ test('type error', function (t) {
         '{ [SyntaxError: whoa] message: \'whoa\', \'a-b\': 5 }'
     ].join(', ') + ' ]');
 });
+
+test('type AssertionError', function (t) {
+    t.plan(3);
+    var err = 'woot', obj2str = Object.prototype.toString;
+    try {
+      assert.deepStrictEqual(true, false);
+    } catch (assertNope) {
+      err = assertNope;
+    }
+    t.equal(err instanceof Error, true);
+    t.equal(obj2str.call(err),    '[object Object]');
```

Comment:
please condense the spaces after the comma to a single space; code should only have semantic, not visual, alignment